content/_index.md

```diff
@@ -225,6 +225,8 @@ Which references VTE upstream: https://gitlab.gnome.org/GNOME/vte/-/issues/253
 
 Currently there is only an old fork (from 2018) that support sixel: https://github.com/saitoha/rxvt-unicode-sixel
```

Comment:
Do you mind editing the "Currently there is only" to "There is"?

Reply:
Ok, will do later

Reply:
Done